external-crates/move/crates/move-analyzer/trace-adapter/tests/references/trace.json

```diff
@@ -0,0 +1,221 @@
+{"version":2}
```

Comment:
Will this test just get picked automatically? Or is this not used anywhere yet?

Reply:
This was a debugging artifact - uncompressed trace from this test. Sorry about being sloppy...